spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1123,6 +1123,12 @@ The typical usage of this property is to be nested in a customized environment <
 +
 Default: false.
 
+spring.cloud.stream.bindingRetryInterval::
+  The interval (seconds) between retrying binding creation when, for example, the binder doesn't support late binding and the broker is down (e.g. Apache Kafka).
+Set to zero to treat such conditions as fatal, preventing the application from starting.
++
+Default: 30
+
 [[binding-properties]]
 === Binding Properties
 
```

Comment:
Since we support late binding from the core framework level with this PR, do we need to streamline the behavior in the rabbit binder? (i.e, all binders reuse this strategy for late binding?)

Reply:
Rabbit's mechanism is more sophisticated (it listens for new connections and will bind faster); I don't think we want to "dumb it down".